Sources/GRPCCore/Configuration/MethodConfiguration.swift

```diff
@@ -0,0 +1,676 @@
+/*
```

Comment:
This isn't actually a new file, it got moved but changed quite substantially, before the configuration only included the timeout and execution policy (i.e. retry/hedging).

Reply:
These changes also have a bit of an impact on `MethodConfigurations` but I'll do those in a separate PR.